src/analysis/locally_convex/bounded.lean

```diff
@@ -0,0 +1,101 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import analysis.normed.normed_field
+import analysis.seminorm
+import topology.algebra.module.basic
+import topology.bornology.basic
+
+/-!
+# Von Neumann Boundedness
+
+This file defines von Neumann bounded sets and proves elementary properties.
+
+## Main declarations
+
+* `is_bounded`: A set `s` is bounded if every neighborhood of zero absorbs `s`.
+
+## Main results
+
+* `bounded_bornology`: The set of bounded sets forms a bornology.
```

Comment:
This is a definition, so I wouldn't label it a "result", but maybe others disagree.

Reply:
true, I will change that